### DISCUSSION TITLE
I need to download a specific directory, but without downloading the whole repository...

### DISCUSSION THREAD

**Discussion by CycloKid** [MrMeowzz/Funkin-MrMeowzz#68] [General]
Do you have the "web" folder that I could download by itself? I've been trying multiple GitHub download directory sites, but none of them work for me.

**Comment by MrMeowzz** [comment]
https://drive.google.com/file/d/1d_m8m1wEW8Nluwewc7Z9lQigOva2Nd_B/view?usp=sharing
